scripts/python/services/test_GithubService.py

```diff
@@ -446,5 +446,19 @@ def test_throws_exception_when_client_throws_exception(self, mock_github_client_
             "test_repository")
 
 
+@patch("gql.transport.aiohttp.AIOHTTPTransport.__new__", new=MagicMock)
+@patch("gql.Client.__new__")
+@patch("github.Github.__new__", new=MagicMock)
+class TestGithubServiceGetTeamIdFromTeamName(unittest.TestCase):
+    def test_calls_downstream_services(self, mock_gql_client):
```

Comment:
light testing here since the function is mainly a thin wrapper for the `gql` client.

This test also silently checks that the gql query is correctly formatted.

It'd be nice to test whether the parameters passed through correctly, although it seemed a bit fiddly to do.

Reply:
The test has no asserts, so what does it test for? Can the test check what is being sent out by the code via a mock object as seen in the [this](https://github.com/ministryofjustice/github-collaborators/blob/eadb59379258f5a58be244a25dd84bad0121530b/spec/helper_module_spec.rb#L283) ruby test example. Or am I mis-reading the test code and it checks it else where?

Reply:
Has the assert on line `456` - just checking that downstream (maybe it's upstream but I always get them confused) are called as a sanity check. When functions do very little, this is usually all you can test 🧪

Reply:
my mistake, sorry, see it now